modules/installation-user-infra-generate-k8s-manifest-ignition.adoc

```diff
@@ -362,12 +362,27 @@ $ openshift-install version
 release image quay.io/openshift-release-dev/ocp-release:4.y.z-x86_64
 ----
 
-.. Locate all `CredentialsRequest` objects in this release image that target the cloud you are deploying on:
+.. Set a `$RELEASE_IMAGE` variable with the release image from your installation file by running the following command:
 +
 [source,terminal]
 ----
-$ oc adm release extract quay.io/openshift-release-dev/ocp-release:4.y.z-x86_64 --credentials-requests --cloud=azure
+$ RELEASE_IMAGE=$(./openshift-install version | awk '/release image/ {print $3}')
 ----
+
+.. Extract the list of `CredentialsRequest` custom resources (CRs) from the {product-title} release image by running the following command:
++
+[source,terminal]
+----
+$ oc adm release extract \
+  --credentials-requests \
+  --to=<path_to_directory_for_credentials_requests> \// <1>
+  --from=$RELEASE_IMAGE \
+  --included \// <2>
```

Comment:
minor nit, if we wanted to order these in processing order, it would be something like:

```console
$ oc adm release extract \
  --from=$RELEASE_IMAGE \
  --credentials-requests \
  --included \// <1>
  --install-config=<path_to_directory_with_installation_configuration>/install-config.yaml \// <2>
  --to=<path_to_directory_for_credentials_requests> // <3>
```

For "take a look at the manifests in `$RELEASE_IMAGE`, grab the CredentialsRequests you expect to be included in clusters like `install-config.yaml`, and drop them into `path_to_directory_for_credentials_requests`".

Reply:
yeah, I like that. Anything that helps it appear more human-readable is an improvement imo